src/main/java/seedu/address/model/person/DisplayedPerson.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Represents person whose records are being displayed.
+ */
+public class DisplayedPerson {
+
+    /** Person to be displayed */
+    private Person person;
+
+    /**
+     * Constructor for DisplayedPerson
+     * @param person person to be displayed.
+     * @param addressBook addressbook containing the person to display.
+     */
+    public DisplayedPerson(Person person, AddressBook addressBook) {
```

Comment:
Perhaps the `addressBook` parameter of this constructor can be removed since it is not used to initialize any fields?

Reply:
Will remove, thanks!